packages/jest-reporters/src/__tests__/summary_reporter.test.js

```diff
@@ -18,6 +18,12 @@ const globalConfig = {
 
 let results = [];
 
+function requireReporter() {
+  jest.isolateModules(() => {
+    SummaryReporter = require('../summary_reporter').default;
```

Comment:
wouldn't it be more straight-forward to call `jest.resetModules` in `beforeEach`?

Reply:
I did it first, but it didn't work. Because 'env' rewrited inside 'test' block. We have to require module after define 'env'.

Reply:
fair enough!